config/interceptor/interceptor.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: keda-http-add-on-interceptor
+  namespace: keda
+  labels:
+    control-plane: interceptor
+    app: keda-http-add-on
+    app.kubernetes.io/name: keda-http-add-on
+    app.kubernetes.io/version: latest
+    app.kubernetes.io/component: interceptor
+    app.kubernetes.io/part-of: keda-http-add-on
+spec:
+  replicas: 1
```

Comment:
Do we control this from code? If so; let's use 3 to be resilient by default

Reply:
We control it yes, we only autogenerated CRDs and RBAC files, so I can change this :)

Reply:
```suggestion
  replicas: 3
```

Reply:
Perfect, thanks!